public/locales/pt/translation.json

```
@@ -1257,6 +1257,10 @@
   "everyService": "Todos os serviços",
   "mobileNetwork": "Rede móvel",
   "enableInterface": "Habilitar esta interface",
+  "disableOutcomeWanInternet": "eliminar o acesso a Internet da CPE, causando transtorno e insatisfação ao cliente, ",
+  "disableOutcomeWanTr069": "desabilitar o gerenciamento TR069 da CPE através do Flashman, o que pode exigir suporte técnico para reconfiguração da CPE, ",
+  "disableOutcomeWanOther": "inativar o serviço associado a esta interface, que poderá causar transtornos e insatisfação ao cliente, ",
+  "wanInterfaceBaseMessage": "Desativar esta interface poderá {{whichMessage}} tem certeza que deseja realizar esta ação?",
```

Como a opção pra TR-069 vai ser capada, talvez limitar isso aqui a uma frase customizada só para o serviço de internet:
```suggestion
  "disableWanInternetMessage": "Desativar esta interface deixará o cliente sem acesso à internet. Tem certeza que deseja desativá-la?",
```
E se for desabilitar uma WAN que só tenha voip / iptv, não precisaria mostrar um alerta